ob1k-cache/src/main/java/com/outbrain/ob1k/cache/metrics/AsyncBulkGetMetrics.java

```diff
@@ -0,0 +1,42 @@
+package com.outbrain.ob1k.cache.metrics;
+
+import com.outbrain.ob1k.concurrent.Try;
+import com.outbrain.swinfra.metrics.api.Counter;
+import com.outbrain.swinfra.metrics.api.MetricFactory;
+import com.outbrain.swinfra.metrics.api.Timer;
+import java.util.Map;
+
+/**
+ * Holds a set of metrics used for measuring get operations.
+ *
+ * @author hunchback
+ */
+public class AsyncBulkGetMetrics extends AsyncOperationMetrics {
```

Comment:
changes:
+ public class AsyncBulkGetMetrics<K> ...
+ for (final K key : keys) ...
+ public void finished(final Try<?> res, ...) ... using "?" so that this is generic and can work for various operations which may have different results passed back by their future.